config/src/main/kotlin/maru/config/Config.kt

```diff
@@ -50,6 +50,13 @@ data class P2P(
   )
 }
 
+data class Sync(
```

Comment:
There's already `SyncingConfig` that I think we should use. But in your case it probably makes sense to introduce a subdivision under the `[syncing]` section, something like `[syncing.downloader]` or `[syncing.block-fetcher]`

Reply:
like that idea